src/views/FindOrder.vue

```diff
@@ -272,51 +216,53 @@ export default defineComponent ({
     IonInfiniteScrollContent,
     IonItem,
     IonLabel,
-    IonList,
-    IonListHeader,
+    IonMenu,
     IonNote,
     IonPage,
-    IonSearchbar,
     IonSelect,
+    IonSelectOption,
+    IonSearchbar,
     IonThumbnail,
     IonTitle,
     IonToggle,
     IonToolbar,
+    OrderFilters
   },
   computed: {
     ...mapGetters({
       orders: 'order/getOrders',
       getProduct: 'product/getProduct',
       currentFacilityId: 'user/getCurrentFacility',
       getProductStock: 'stock/getProductStock',
-      isScrollable: 'order/isScrollable'
+      isScrollable: 'order/isScrollable',
+      query: 'order/getOrderQuery'
     })
   },
+  data() {
+    return {
+      shippingMethodOptions: ['any'],
+      orderStatusOptions: ['any'],
+      sort: 'orderDate desc',
+      showOrderItems: true,
+      poIds: {} as any
+    }
+  },
   methods: {
-    async getOrders(vSize?: any, vIndex?: any){
+    async sortOrders(value: string) {
+      this.sort = value
+      await this.store.dispatch('order/updateSortOption', this.sort)
+      this.getOrders();
```

Comment:
I think we could this in updateSort action

Reply:
Moved the getOrders call to actions.